utilities/geodb/geodb_impl.cc

```diff
@@ -284,10 +284,11 @@ std::string GeoDBImpl::MakeKey2(Slice id) {
 std::string GeoDBImpl::MakeKey1Prefix(std::string quadkey,
                                       Slice id) {
   std::string key = "p:";
-  key.reserve(3 + quadkey.size() + id.size());
+  key.reserve(4 + quadkey.size() + id.size());
   key.append(quadkey);
   key.append(":");
   key.append(id.ToString());
+  key.append(":");
```

Comment:
why do you need to append an extra ":" at the end of the prefix?

Reply:
Append an extra ":" to avoid seek an error key when it has the same prefix with the correct one.
For example, as descripted in hey https://github.com/acelyc111/rocksdb/blob/master/utilities/geodb/geodb_test.cc#L134, there are two KVs, 
`key1("12"), pos1(40.00001, 116.00001)` 
and
 `key2("123"), pos2(40.00002, 116.00002)`
, they are close to each other and have the **same quadkey** 
`13210010233131212010322`.
If we want to `GetById("12")`, we will get quadkey `13210010233131212010322` in the 1st stage, and we will got the incorrect `key2("123")` in the 2nd stage, because we use `Seek`, and the incorrect combine key `p:13210010233131212010322:123` is before correct key `p:13210010233131212010322:12:` in db.